jenkins_scripts/ami_jenkins_server_setup.sh

```diff
@@ -1,7 +1,8 @@
 sudo wget -O /etc/yum.repos.d/jenkins.repo http://pkg.jenkins-ci.org/redhat/jenkins.repo
 sudo rpm --import https://jenkins-ci.org/redhat/jenkins-ci.org.key
 sudo yum install -y jenkins java git docker python3 python3-pip \
-    python3-devel libmemcached gcc libmemcached-devel zlib-devel
+    python3-devel libmemcached gcc libmemcached-devel zlib-devel \
+    postgresql-devel
```

Comment:
I'm curious why this didn't hit my Docker build. Let me test that again…

Reply:
I tested it with Docker locally too, and it was fine. It's on crowd-dev now too.